ffwd-client/src/main/java/com/spotify/ffwd/v1/Metric.java

```diff
@@ -18,24 +18,6 @@
  * -/-/-
  */
 
-/*
- * FastForward Client
```

Comment:
I believe this needs to be at the top of all our open source files, would be good to add back

Reply:
Not sure what's happening, but if you check the actual file you will see  the  header.